### DISCUSSION TITLE
Off site click tracking: domain report?

### DISCUSSION THREAD

**Discussion by freitasm** [plausible/analytics#821] [Anything Else]
I like how click tracking can be a goal and the quick access to reporting.

Would there be a way to have an option to report on the destination domain instead of full URL?

**Comment by Vigasaurus** [comment]
Not directly with the outbound-links js script, but you could load the normal script and emulate the javascript that the outbound links script uses yourself. 

https://github.com/plausible/analytics/blob/master/tracker/src/plausible.js#L74 (just swap the `link.href` to `link.hostname`)

**Reply by freitasm** [reply]
Thanks... That's an interesting idea - lose the whole URL but track per domain. You mean I could just go back to using the standard script and locally implement handleOutbound and registerOutboundLinkEvents() to have the same functionality?

**Reply by Vigasaurus** [reply]
You could, yeah. And using the hostname is just an example, you could track any amount of other custom props there, be it hostname, the full path, both, or others. 

The important bit there is just line 84, you could even just do that in normal JS button functions etc, you don't have to use the same event listener method, necessarily.

**Reply by freitasm** [reply]
Thanks. Will test this on dev subdomain.
